src/qt/utilitydialog.cpp

```diff
@@ -20,6 +20,9 @@
 #include "clientversion.h"
 #include "init.h"
 #include "util.h"
+#ifdef ENABLE_WALLET
```

Comment:
I'm not very familiar with the qt code. I see that there are already plenty of `#ifdef ENABLE_WALLET` in the qt code already. Is there something special about utilitydialog.cpp ?

I'm very happy to take an alternative approach if you have suggestions.